### PR TITLE
Update several tags in common and frontend @beta to public

### PR DESCRIPTION
**Pull request by kabentley**
- Tween api
- IModelConnectionProps
- several types for Ipc, most still in @beta

### CODE REVIEW

File: core/common/src/IModel.ts [hunk 1]
```diff
@@ -123,8 +127,8 @@ export interface SnapshotOpenOptions extends IModelEncryptionProps, OpenDbKey {
   autoUploadBlocks?: boolean;
 }
 
-/** Options that can be supplied when opening an existing StandaloneDb.
- * @beta
+/** Options supplied to open an existing StandaloneDb.
```
Comment (pmconne) [inline core/common/src/IModel.ts:130]: StandaloneDb is internal. Change to "standalone BriefcaseConnection"?

**Review by pmconne — CHANGES_REQUESTED** [review]

File: core/backend/src/IModelDb.ts [hunk 2]
```diff
@@ -2651,9 +2652,8 @@ export class SnapshotDb extends IModelDb {
  * - may be opened without supplying any user credentials
  * - may be opened read/write
  * - may optionally support undo/redo via [[TxmManager]]
- * - cannot apply a changeset to nor generate a changesets
- * - are only available to authorized applications
- * @internal
+ * - cannot apply a changeset to nor generate a changesets (since there is no timeline from which to get/push changesets)
+ * @public
```
Comment (pmconne) [inline core/backend/src/IModelDb.ts:2656]: Can you add something like `@see [BriefcaseConnection.openStandalone]($frontend) to open a StandaloneDb on the frontend`? The relationship between StandaloneDb and BriefcaseConnection is not obvious.

File: core/common/src/IModel.ts [hunk 3]
```diff
@@ -127,7 +127,7 @@ export interface SnapshotOpenOptions extends IModelEncryptionProps, OpenDbKey {
   autoUploadBlocks?: boolean;
 }
 
-/** Options supplied to open an existing StandaloneDb.
+/** Options supplied to open a StandaloneDb.
```
Comment (pmconne) [inline core/common/src/IModel.ts:130]: Can you turn `StandaloneDb` into a link - `[StandaloneDb]($backend)`?

File: core/backend/src/IModelDb.ts (hunk 2 above)
Comment (pmconne) [reply core/backend/src/IModelDb.ts:2656]: However `openStandalone` is currently `internal`.

Comment (kabentley) [reply core/backend/src/IModelDb.ts:2656]: made it @public

File: core/common/src/IModel.ts (hunk 3 above)
Comment (kabentley) [reply core/common/src/IModel.ts]: done

**Review by pmconne — APPROVED** [review]

File: core/common/src/IModel.ts (hunk 1 above)
Comment (calebmshafer) [reply core/common/src/IModel.ts]: @kabentley I thought `StandaloneDb`s were no longer going to be used outside of tests. How would someone use them in a regular workflow?

**Pull request merged**